packages/cli/src/commands/identify.js

```diff
@@ -0,0 +1,14 @@
+/* global process */
+import os from 'os';
+import { E } from '@endo/far';
+import { withEndoBootstrap } from '../context.js';
+
+export const identify = async ({ cancel, cancelled, sockPath, namePath }) =>
+  withEndoBootstrap({ os, process }, async ({ bootstrap }) => {
+    const defaultHostFormulaIdentifier = 'host';
+    const formulaId = await E(bootstrap).identifyFrom(
+      defaultHostFormulaIdentifier,
+      namePath,
+    );
+    console.log(formulaId);
+  });
```

Comment:
This is very close to what I imagined and a step in the right direction.

High level:
- We need an API and a command that can reveal the formula type. I had envisioned that `endo identify` would do that, but perhaps that should have a name like `describe`. I think `endo describe` should also be able to print the nonce/swissnum/identifier like `endo describe --id <name>`
- Whatever API and command reveals the nonce/swissnum/identifier for a pet name path should probably be producing a URL so maybe that should be called `endo locate <pet-name-path>` and produce the URL by default, including the connection hints for the listening networks. Maybe `endo locate --json <path>` gives you the JSON representation. Maybe `endo locate --id <path>` trims that back to just the nonce.
- Maybe `locate` and `describe` are the same command or two commands with the same behavior but different defaults.

Nits:
- Let’s get rid of `cancel`, `cancelled`, and `sockPath` here and where this refactor-cruft was copied from.
- `host` is not a formula identifier anymore. We should always start looking stuff up at `E(bootstrap).host()`.
- If we move some functions from the endo bootstrap object to host objects, maybe we can just present the primary host facet as the bootstrap object.
- Maybe we should use ``process.stdout.write(`${id}\n`)`` instead of `console.log` just to break the bad habit of confusing the debugger for an output printer. ymmv.